l5kit/l5kit/data/map_api.py

```diff
@@ -134,6 +144,92 @@ def get_lane_coords(self, element_id: str) -> dict:
 
         return {"xyz_left": xyz_left, "xyz_right": xyz_right}
 
+    @staticmethod
+    def interpolate(xyz: np.ndarray, cum_dist: np.ndarray, step: float, method: InterpolationMethod) -> np.ndarray:
+        """
+        Interpolate points based on cumulative distances from the first one. Two modes are available:
+        INTER_METER: interpolate using step as a meter value over cumulative distances (variable len result)
+        INTER_ENSURE_LEN: interpolate using a variable step such that we always get step values
+        Args:
+            xyz (np.ndarray): XYZ coords
+            cum_dist (np.ndarray): distances from the first coords of xyz. Same length as xyz
+            step (float): param for the interpolation
+            method (InterpolationMethod): method to use to interpolate
+
+        Returns:
+            np.ndarray: the new interpolated coordinates
```

Comment:
I think we should start migrating the doc format to the same that avjadoo uses:
```
:param xyz: etc
:param cum_dist: etc
```
Just to have a single unified documentation format.

Reply:
so reST and not Google then...sigh I love the Google's one xD

Reply:
I will leave it as it is for now. I think we should aim for a single PR to change them all. I will start working on that

Reply:
👍